cypress/integration/analytics.test.js

```diff
@@ -190,3 +190,21 @@ describe('view_item_list event', () => {
     })
   })
 })
+
+describe('search event', () => {
+  beforeEach(() => {
+    cy.clearIDB()
+  })
```

Comment:
I would suggest moving this to the global scope, just as I did at https://github.com/vtex-sites/base.store/pull/203/commits/d68507d9f9426ac65987dc211eada8900327869c. Just to avoid merging conflicts and/or duplicate code when both get merged.

I think it's repetitive/pointless if we repeat the `cy.clearIDB()` inside each and every `describe` statement. Since we do not want to share state between tests, better move it to the global scope already and make this IDB clearance a default for every new test. This might even make this less error-prone, e.g. where someone forgets to clear the IDB before writing a new tests. 

What do you think?

Reply:
Makes total sense, Heitor. Thanks.